yarn.lock

```diff
@@ -743,6 +743,41 @@
   resolved "https://registry.yarnpkg.com/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz#75a2e8b51cb758a7553d6804a5932d7aace75c39"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
+"@endo/env-options@^0.1.4":
```

Comment:
Don't worry yet about these yarn.lock changes. After the current endo-release-agoric_sdk-upgrade dance settles down and I rebase this PR, I see what fresh changes it causes to yarn.lock.

Reply:
I separated the yarn.lock changes into a separate commit. Since this PR does involve package.json dependency changes, I doubt I could merge without some yarn.lock changes. However, neither should I commit these as long as https://github.com/endojs/endo/pull/1904 needs investigation.

Reply:
Merging #1904 and rebasing this PR caused this PR to drop its yarn.lock changes. Redoing `yarn` did not create any changes. So this PR is now good to go. Thanks!